[21:00:39] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:00:39] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:00:39] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:00:39] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:00:39] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:00:39] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:00:39] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:00:39] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:00:39] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:00:39] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:05:40] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:05:40] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:05:40] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:05:40] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:05:40] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:05:40] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:05:40] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:05:40] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:05:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:05:40] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:10:40] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:10:40] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:10:40] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:10:40] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:10:40] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:10:40] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:10:40] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:10:40] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:10:40] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:10:40] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:15:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:15:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:15:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:15:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:15:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:15:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:15:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:15:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:15:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:15:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:20:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:20:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:20:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:20:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:20:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:20:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:20:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:20:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:20:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:20:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:25:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:25:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:25:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:25:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:25:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:25:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:25:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:25:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:25:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:25:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:30:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:30:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:30:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:30:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:30:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:30:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:30:41] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:30:41] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:30:41] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:30:41] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:35:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:35:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:35:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:35:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:35:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:35:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:35:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:35:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:35:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:35:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:40:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:40:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:40:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:40:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:40:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:40:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:40:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:40:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:40:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:40:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:45:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:45:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:45:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:45:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:45:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:45:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:45:42] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:45:42] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:45:42] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:45:42] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:50:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:50:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:50:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:50:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:50:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:50:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:50:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:50:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:50:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:50:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:55:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:55:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:55:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:55:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:55:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:55:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[21:55:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[21:55:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[21:55:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[21:55:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:00:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:00:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:00:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:00:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:00:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:00:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:00:43] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:00:43] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:00:43] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:00:43] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:05:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:05:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:05:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:05:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:05:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:05:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:05:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:05:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:05:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:05:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:10:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:10:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:10:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:10:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:10:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:10:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:10:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:10:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:10:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:10:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:15:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:15:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:15:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:15:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:15:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:15:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:15:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:15:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:15:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:15:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:20:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:20:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:20:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:20:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:20:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:20:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:20:46] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:20:46] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:20:46] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:20:46] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:25:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:25:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:25:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:25:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:25:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:25:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:25:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:25:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:25:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:25:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:30:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:30:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:30:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:30:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:30:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:30:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:30:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:30:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:30:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:30:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:35:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:35:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:35:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:35:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:35:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:35:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:35:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:35:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:35:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:35:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:40:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:40:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:40:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:40:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:40:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:40:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:40:47] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:40:47] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:40:47] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:40:47] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:45:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:45:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:45:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:45:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:45:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:45:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:45:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:45:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:45:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:45:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:50:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:50:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:50:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:50:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:50:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:50:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:50:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:50:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:50:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:50:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:55:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:55:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:55:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:55:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:55:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:55:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[22:55:48] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[22:55:48] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[22:55:48] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[22:55:48] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:00:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:00:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:00:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:00:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:00:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:00:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:00:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:00:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:00:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:00:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:05:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:05:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:05:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:05:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:05:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:05:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:05:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:05:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:05:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:05:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:10:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:10:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:10:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:10:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:10:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:10:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:10:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:10:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:10:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:10:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:15:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:15:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:15:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:15:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:15:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:15:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:15:49] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:15:49] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:15:49] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:15:49] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:20:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:20:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:20:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:20:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:20:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:20:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:20:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:20:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:20:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:20:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:25:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:25:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:25:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:25:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:25:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:25:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:25:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:25:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:25:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:25:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:30:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:30:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:30:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:30:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:30:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:30:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:30:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:30:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:30:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:30:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:35:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world is being accessed from another location, aborting
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:35:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:35:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:35:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_nether is being accessed from another location, aborting
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:35:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:35:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[23:35:50] [Server thread/WARN]: net.minecraft.server.v1_12_R1.ExceptionWorldConflict: The save for world located at .\world_the_end is being accessed from another location, aborting
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldNBTStorage.checkSession(WorldNBTStorage.java:78)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.World.checkSession(World.java:2820)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.a(WorldServer.java:1077)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.WorldServer.save(WorldServer.java:1047)
[23:35:50] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_12_R1.CraftWorld.save(CraftWorld.java:748)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:703)
[23:35:50] [Server thread/WARN]: 	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[23:35:50] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
